<launch>
  <!-- run simple_topic_publisher.py from my_package w=640 h=480 -->
 <!--  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" /> 
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node> -->

  <node pkg="my_package" type="laser_alignment.py" name="Drone_laser_alignment"  output="screen">
  </node>
</launch>
